tests/pytests/test_newhorizons_drivers.py

```diff
@@ -57,6 +58,17 @@ def test_nhleisa_load(test_kernels, image):
     comparison = compare_dicts(isd_obj, compare_isd)
     assert comparison == []
 
+# Test load of nh leisa labels
+@pytest.mark.parametrize("image", ['mc3_0034948318_0x536_sci_1'])
+def test_nhmvictdi_load(test_kernels, image):
+    label_file = get_image_label(image, 'isis')
+    isd_str = ale.loads(label_file, props={'kernels': test_kernels[image]})
+    compare_isd = image_dict[image]
+    isd_obj = json.loads(isd_str)
+    print(json.dumps(isd_obj, indent=2))
```

Comment:
Are these debug prints or is this part of the test?  I remember seeing in the file previously.

Reply:
Largely just debug prints that can be removed